Cargo.toml

```diff
@@ -7,7 +7,7 @@ description = "Elastic vector backed by fixed size array"
 repository = "https://github.com/debris/elastic-array"
 
 [dependencies]
-heapsize = { version = "0.3", features = ["unstable"] }
+heapsize = "0.4"
```

Comment:
having made a quick glance to the heapsize repo, the `unstable` feature is left for backward compatibility only (they probably used it for custom `derive`, but now they use procedural macros which are stable)